docs/design/ovs-pipeline.md

```diff
@@ -193,21 +200,24 @@ This table is used to determine which "category" of traffic (tunnel, local
 gateway or local Pod) the packet belongs to. This is done by matching on the
 ingress port for the packet. The appropriate value is then written to bits
 [0..15] of the NXM_NX_REG0 register: 0 for tunnel, 1 for local gateway and 2 for
-local Pod. This information is used by matches in subsequent tables.
+local Pod. This information is used by matches in subsequent tables. For a
+packet received from the tunnel port, bit 19 of the NXM_NX_REG0 is set to 1, to
+indicate MAC rewritting should be performed for the packet in the
+[L3ForwardingTable].
 
 If you dump the flows for this table, you may see the following:
 
 ```text
 1. table=0, priority=200,in_port=antrea-gw0 actions=load:0x1->NXM_NX_REG0[0..15],goto_table:10
-2. table=0, priority=200,in_port=antrea-tun0 actions=load:0->NXM_NX_REG0[0..15],goto_table:30
+2. table=0, priority=200,in_port=antrea-tun0 actions=load:0->NXM_NX_REG0[0..15],load:0x1->NXM_NX_REG0[19],goto_table:30
 3. table=0, priority=190,in_port="coredns5-8ec607" actions=load:0x2->NXM_NX_REG0[0..15],goto_table:10
 4. table=0, priority=190,in_port="coredns5-9d9530" actions=load:0x2->NXM_NX_REG0[0..15],goto_table:10
 5. table=0, priority=0 actions=drop
 ```
 
 Flow 1 is for traffic coming in on the local gateway. Flow 2 is for traffic
 coming in through an overlay tunnel (i.e. from another Node). The next two
-flows (3 and 4) are for local Pods (in this case Pods from the coredns
+flows (3 and 4) are for local Pods (in this case Pods from the CoreDNS
```

Comment:
maybe the intention was to use the actual name of the K8s Deployment resource, in which case it should be: `coredns` Deployment?

Reply:
I feel CoreDNS might be easier for readers, but let me know what you suggest.